test/TestProjects/MgmtMockAndSample/src/Generated/ManagedHsmResource.cs

```diff
@@ -431,7 +431,7 @@ public virtual ArmOperation<ManagedHsmResource> Update(WaitUntil waitUntil, Mana
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>MHSMPrivateLinkResources_ListByMhsmResource</description>
```

Comment:
The new operation id is what defined in swagger: https://github.com/Azure/autorest.csharp/blob/371c2a2029ae8a2baef218b9b55798433f4b7b75/test/TestProjects/MgmtMockAndSample/specification/mockSwagger/managedHsm.json#L736